libs/application-generic/src/usecases/tier-restrictions-validate/tier-restrictions-validate.usecase.ts

```diff
@@ -142,6 +166,10 @@ const isCronExpression = (cron: string) => {
 };
 
 const isRegularDeferAction = (command: TierRestrictionsValidateCommand) => {
+  if (command.deferDurationMs) {
```

Comment:
a bug we had, where we did not check the deferDurationMs that the worker was passing.